fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1985,12 +1985,17 @@ public Void visitDictionaryGetExpr(DictionaryGetExpr node, Scope context) {
             List<String> dictionaryKeys = dictionary.getKeys();
             int dictionaryKeysSize = dictionaryKeys.size();
             int paramDictionaryKeysSize = params.size() - 1;
-            if (paramDictionaryKeysSize != dictionaryKeysSize) {
+            if (!(paramDictionaryKeysSize == dictionaryKeysSize || paramDictionaryKeysSize == dictionaryKeysSize + 1)) {
                 throw new SemanticException("dictionary: " + dictionaryName + " has keys size: " +
```

Comment:
should improve the error message, better to add expected param size

Reply:
Updated